MANUAL.txt

```diff
@@ -3251,7 +3260,7 @@ If you just want a regular inline image, just make sure it is not
 the only thing in the paragraph. One way to do this is to insert a
 nonbreaking space after the image:
 
-    ![This image won't be a figure](/url/of/image.png)\ 
```

Comment:
Eeek!  Very sorry, I can't believe I didn't see this!  The new diff against master should be

```diff
diff --git a/MANUAL.txt b/MANUAL.txt
index 7e0bdfc..1b7454d 100644
--- a/MANUAL.txt
+++ b/MANUAL.txt
@@ -2518,6 +2518,15 @@ For headerless tables, the colons go on the top line instead:
     | Right         | Left          | Centered           |
     +---------------+---------------+--------------------+
 
+##### Grid Table Limitations #####
+
+Pandoc does not support grid tables with row spans (variable number
+of columns in a given row). All grid tables must have the same
+number of columns in each row.  For example, the Docutils
+[sample grid tables] will not render as expected with Pandoc.
+
+[sample grid tables]: http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#grid-tables
+
 
 #### Extension: `pipe_tables` ####
 

```